android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java

```diff
@@ -281,6 +332,287 @@ public void connectAudio() {
+      return;
+    }
+
+    final InMeetingService inMeetingService = zoomSDK.getInMeetingService();
+
+    promise.resolve(inMeetingService.isMeetingConnected());
+  }
+
+  @ReactMethod
+  public void isMeetingHost(Promise promise) {
+    WritableArray rnUserList = Arguments.createArray();
+    ZoomSDK zoomSDK = ZoomSDK.getInstance();
+
+    if (!zoomSDK.isInitialized()) {
+      promise.resolve(false);
+      return;
+    }
+
+    final InMeetingService inMeetingService = zoomSDK.getInMeetingService();
+
+    promise.resolve(inMeetingService.isMeetingHost());
+  }
+
+  @ReactMethod
+  public void getInMeetingUserList(Promise promise) {
+    WritableArray rnUserList = Arguments.createArray();
+    ZoomSDK zoomSDK = ZoomSDK.getInstance();
+
+    if (!zoomSDK.isInitialized()) {
+      promise.resolve(rnUserList);
+      return;
+    }
+
+    final InMeetingService inMeetingService = zoomSDK.getInMeetingService();
+
+    final List<Long> userList = inMeetingService.getInMeetingUserList();
+
+    for (final Long userId : userList) {
+      rnUserList.pushString(userId.toString());
+    }
+
+    promise.resolve(rnUserList);
+  }
+
+  @ReactMethod
+  public void muteMyVideo(final boolean muted, Promise promise) {
+    ZoomSDK zoomSDK = ZoomSDK.getInstance();
+
+    if (!zoomSDK.isInitialized()) {
+      return;
+    }
+
+    final InMeetingService inMeetingService = zoomSDK.getInMeetingService();
+
+    final InMeetingVideoController videoController = inMeetingService.getInMeetingVideoController();
+
+    MobileRTCSDKError result = videoController.muteMyVideo(muted);
+
+    if (result == MobileRTCSDKError.SDKERR_SUCCESS) {
+      promise.resolve(null);
+    } else {
+      promise.reject("ERR_ZOOM_MEETING", "Mute my video error, status: " + result.name());
+    }
+  }
+
+  @ReactMethod
+  public void rotateMyVideo(final int rotation, Promise promise) {
+    ZoomSDK zoomSDK = ZoomSDK.getInstance();
+
+    if (!zoomSDK.isInitialized()) {
+      return;
+    }
+
+    final InMeetingService inMeetingService = zoomSDK.getInMeetingService();
+
+    final InMeetingVideoController videoController = inMeetingService.getInMeetingVideoController();
+
+    if (videoController.rotateMyVideo(rotation)) {
+      promise.resolve(null);
+    } else {
+      promise.reject("ERR_ZOOM_MEETING", "Error: Rotate video failed");
+    }
+  }
+
+  @ReactMethod
+  public void muteMyAudio(final boolean muted, Promise promise) {
+    ZoomSDK zoomSDK = ZoomSDK.getInstance();
+
+    if (!zoomSDK.isInitialized()) {
+      return;
+    }
+
+    final InMeetingService inMeetingService = zoomSDK.getInMeetingService();
+
+    final InMeetingAudioController audioController = inMeetingService.getInMeetingAudioController();
+
+    MobileRTCSDKError result = audioController.muteMyAudio(muted);
+
+    if (result == MobileRTCSDKError.SDKERR_SUCCESS) {
+      promise.resolve(null);
+    } else {
+      promise.reject("ERR_ZOOM_MEETING", "Mute myself error, status: " + result.name());
```

Comment:
to be in sync with `Mute my video`
```suggestion
      promise.reject("ERR_ZOOM_MEETING", "Mute my audio error, status: " + result.name());
```